src/Hubbup.Web/DataSources/GitHubDataSource.cs

```diff
@@ -79,13 +91,17 @@ public GitHubDataSource(ILogger<GitHubDataSource> logger)
                 data = result.Data;
 
                 // Add rate limit info
-                _logger.LogTrace("Request completed, consumed {cost} of rate limit {limit}. Remaining: {remaining}, resets at {resetAt}",
+                _logger.LogTrace("Request completed, consumed {Cost} of rate limit {Limit}. Remaining: {Remaining}, resets at {ResetAt}",
                     data.RateLimit.Cost,
                     data.RateLimit.Limit,
                     data.RateLimit.Remaining,
                     data.RateLimit.ResetAt);
                 rateLimitInfo = RateLimitInfo.Add(rateLimitInfo, data.RateLimit);
 
+                _metricsService.Record("GitHubDataSource:QueryRateLimitCost", data.RateLimit.Cost);
```

Comment:
We kind of talked about this in person the other day - it's a bit dubious to track this without including the user's account name (or id, or whatever), because it's a per-GitHub-account metric. But whatever.

Reply:
Good point, I can look at adding a username tag in.